### PR TITLE
Initial Release Candidate

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@richardkchapman - Can you review for conventions and licenses etc. (I am not expecting this to pulled just yet).

Also I need to improve the readme and may add some travis sanity tests.

**Comment by richardkchapman** [comment]
Should also create a PR against https://github.com/hpcc-systems/ecl-bundles to add to the "supported bundles" list

**Comment by GordonSmith** [comment]
@richardkchapman Switched sample data.

File: Visualizer.ecl [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+    IMPORT Std;
+
+    EXPORT Bundle := MODULE(Std.BundleBase)
+        EXPORT Name := 'Visualizer';
+        EXPORT Description := 'ECL Visualization Bundle';
+        EXPORT Authors := ['HPCC Systems'];
+        EXPORT License := 'http://www.apache.org/licenses/LICENSE-2.0';
+        EXPORT Copyright := 'Copyright (C) 2016 HPCC Systems';
+        EXPORT DependsOn := [];
+        EXPORT Version := '1.0.0-rc1';
```
Comment (richardkchapman) [inline Visualizer.ecl]: should be updated

File: res/index.js [hunk 2]
```diff
@@ -0,0 +1,319 @@
+function requireApp(require, callback) {
+    "use strict";
+    require(["d3", "src/composite/Dermatology", "src/common/Widget", "src/other/ESP", "src/layout/Grid", "src/other/Persist", "src/common/Utility", "src/composite/MegaChart", "src/form/Button"], function (d3, Dermatology, Widget, ESP, Grid, Persist, Utility, MegaChart, Button) {
+        Dermatology = Dermatology.Dermatology || Dermatology;
+        Widget = Widget.Widget || Widget;
+        var __HPCC_VISUALIZATION = "__hpcc_visualization";
+        var HPCC_VIZBUNDLE = "HPCC-VizBundle";
```
Comment (richardkchapman) [inline res/index.js]: Is this right?

**Comment by GordonSmith** [comment]
@richardkchapman changes made.

**Pull request merged**